test/Macros/macro_expand_codeitems.swift

```diff
@@ -0,0 +1,34 @@
+// REQUIRES: executable_test
```

Comment:
```suggestion
// REQUIRES: swift_swift_parser, executable_test
```

Reply:
Gah, thanks... I swear I was going to add it after seeing your message 😅